applicationmetadata/service.go

```diff
@@ -0,0 +1,17 @@
+package applicationmetadata
+
+import (
+	"github.com/golang/protobuf/proto"
+)
+
+//go:generate protoc --go_out=. ./message.proto
+
+func Handle(payload []byte) (*Message, error) {
```

Comment:
Too many `handle` methods we have :D 

```suggestion
func Unmarshal(payload []byte) (*Message, error) {
```